### DISCUSSION TITLE
Prompt Show-ADTInstallationWelcome from SYSTEM context to user context

### DISCUSSION THREAD

**Discussion by fethiarras** [PSAppDeployToolkit/PSAppDeployToolkit#1195] [Q&A] [answered]
Hi,

Is there a way to display the Show-ADTInstallationWelcome command on user context from SYSTEM context without psexec and i try Service_UI it don't work ? Is there a built-in function with the new PSADT upgrade ?

Thanks,

**Comment by DanGough** [comment, marked as answer]
No, but this is planned for a future release, perhaps 4.1 a few months down the line. The ideal situation is that the main toolkit runs in session 0, but launches and communicates with UI processes running as the user.
